arbnode/batch_poster.go

```diff
@@ -622,6 +629,23 @@ func (b *BatchPoster) Start(ctxIn context.Context) {
 	b.redisLock.Start(ctxIn)
 	b.StopWaiter.Start(ctxIn, b)
 	b.CallIteratively(func(ctx context.Context) time.Duration {
+		var err error
+		if common.HexToAddress(b.config().GasRefunderAddress) != (common.Address{}) {
+			var gasRefunderBalance *big.Int
+			gasRefunderBalance, err = b.l1Reader.Client().BalanceAt(ctx, common.HexToAddress(b.config().GasRefunderAddress), nil)
```

Comment:
This is minor, but there's no need to forward declare the variables either here or in the validator. You can just do `gasRefunderBalance, err :=`. Go is completely fine shadowing a variable from the outer scope too.

Reply:
done.